src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -16,7 +17,7 @@
     <StringListProperty Name="ProjectTypeGuids" Visible="False" />
     <StringProperty Name="ProjectGuid" Visible="False">
         <StringProperty.DataSource>
-            <DataSource Persistence="ProjectFile" Label="Globals" HasConfigurationCondition="False" />
+            <DataSource Persistence="Implicit" Label="Globals" HasConfigurationCondition="False" />
```

Comment:
This is a cool approach, can we make this name a little clear that's its writing the ProjectFile as opposed to say the UserFile. ie Can we call this ImplictProjectFile or something?